### PR TITLE
Update readme (also some updated migration generators)

### PR DESCRIPTION
**Pull request by amclelland**

### CODE REVIEW

**Comment by artfuldodger** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -40,7 +38,40 @@ and the requisite tables will be created.
 
 ## Usage
 
-TODO: Write usage instructions here
+Add your Pivotal Tracker API token to your secrets.yml:
+
+`tracker_api_token: <%= ENV["PIVOTAL_TRACKER_API_TOKEN"] %>`
+
+Create a new rake task called `import.rake`
```
Comment (jakesutton) [inline README.md:45]: Can the Gem not include executable rake files?

Comment (jakesutton) [inline README.md:45]: Or at least a generator or install script... ¯\(°_°)/¯

**Comment by jakesutton** [comment]
:+1:

File: README.md (hunk 1 above)
Comment (amclelland) [inline README.md:45]: Hmm interesting, I'll look into that. That would be useful. My only concern is making sure the user can specify the interval

Comment (jakesutton) [inline README.md:45]: If you do figure it out, be sure to namespace the tasks appropriately. `rake active_pivot:import:initial 8/12/2012`

[Here's a post about passing args to a rake task](http://thepolymathlab.com/4-ways-to-pass-arguments-to-a-rake-task). (My example would be method 3.)

Comment (amclelland) [inline README.md:45]: Okay, finished this up and PR'd it!
https://github.com/foraker/active_pivot/pull/2

**Pull request merged**